var/spack/repos/builtin/packages/pdc/package.py

```diff
@@ -14,15 +14,25 @@ class Pdc(CMakePackage):
     metadata operations to find data objects."""
 
     homepage = "https://pdc.readthedocs.io/en/latest/"
-    url      = "https://github.com/hpc-io/pdc/archive/refs/tags/0.1.tar.gz"
+    url      = "https://github.com/hpc-io/pdc/archive/refs/tags/0.3.tar.gz"
+    git      = "https://github.com/hpc-io/pdc.git"
 
     maintainers = ['houjun', 'sbyna']
 
+    version('0.3', sha256='14a3abd5e1e604f9527105709fca545bcdebe51abd2b89884db74d48a38b5443')
+    version('0.2', sha256='2829e74da227913a1a8e3e4f64e8f422ab9c0a049f8d73ff7b6ca12463959f8b')
     version('0.1', sha256='01b4207ecf71594a7f339c315f2869b3fa8fbd34b085963dc4c1bdc5b66bb93e')
 
+    version('stable', branch='stable')
+    version('develop', branch='develop')
+
+    variant('fabrics', default=False, description='Build support for sockets, TCP, UDP, and RXM fabrics')
+
     conflicts('%clang')
     depends_on('libfabric')
+    depends_on('libfabric fabrics=sockets,tcp,udp,rxm', when='+fabrics')
     depends_on('mercury')
+    depends_on('cmake')
```

Comment:
The `cmake` build dependency is automatically included by inheriting from `CMakePackage`. Also that way `cmake` will be a `build`-only dependency (the default is to link to cmake and include it at runtime, neither of which are good).

Reply:
Got it!